src/views/SsoSettings.vue

```diff
@@ -0,0 +1,483 @@
+                           data-qa="tenant-id-input"
+                           title="Tenant Id"
+                           placeholder="Tenant Id"
+                           :disabled="!ssoData.enabled">
+                  </div>
+                </div>
+                <div class="field">
+                  <label class="label is-small">
+                    Entity id
+                  </label>
+                  <div class="control suffix">
+                    <input v-model="ssoData.params.entityId"
+                           class="input is-small"
+                           data-qa="entity-id-input"
+                           title="Entity Id"
+                           placeholder="Entity Id"
+                           :disabled="!ssoData.enabled">
+                  </div>
+                </div>
+                <!-- TODO: v-model="localDoc.description" -->
+                <div class="field textarea-field">
+                  <label class="label is-small">x509cert</label>
+                  <div class="control">
+                        <textarea v-model="ssoData.params.x509cert"
+                                  class="is-small textarea"
+                                  data-qa="x509cert-input"
+                                  title="x509certificate"
+                                  rows="7"
+                                  :disabled="!ssoData.enabled">
+                        </textarea>
+                  </div>
+                </div>
+                <div class="field">
+                  <label class="label is-small">
+                    Login URL
+                  </label>
+                  <div class="control suffix">
+                    <input v-model="ssoData.params.login_url"
+                           class="input is-small"
+                           data-qa="login-url-input"
+                           title="Login URL"
+                           placeholder="Login URL"
+                           :disabled="!ssoData.enabled">
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div v-else class="content no-data-wrapper">
+            <div v-if="loadingCounter > 0">
+              <button class="button is-outlined is-text is-small is-loading value-loading">
+                Loading
+              </button>
+            </div>
+            <div v-else
+                class="no-data-message">
+              No data found.
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </template>
+
+<script lang="ts">
+import {defineComponent} from 'vue'
+import {useBranchesStore} from '@/stores/BranchesStore'
+import {Sso} from '@/types'
+import {mapStores} from 'pinia'
+import RequestsUtils from '@/assets/RequestsUtils'
+import DatasetsUtils from '@/assets/DatasetsUtils'
+
+export default defineComponent({
+  name: 'SsoSettings',
+  data() {
+    const provider = ['azure', 'okta']
```

Comment:
It's a list of providers, not a single provider

```suggestion
    const providers = ['azure', 'okta']
```

Reply:
Done